lib/controllers/ServiceFabrikAdminController.js

```diff
@@ -24,24 +24,42 @@ class ServiceFabrikAdminController extends BaseController {
   }
 
   updateDeployment(req, res) {
+    const self = this;
     const redirect_uri = _.get(req.query, 'redirect_uri', '/admin/deployments/outdated');
-    return this.fabrik
-      .createOperation('update', {
-        deployment: req.params.name,
-        username: req.user.name,
-        arguments: req.body
-      })
-      .invoke()
-      .then(body => {
-        res.format({
-          html: () => res
-            .redirect(303, redirect_uri),
-          default: () => res
-            .status(200)
-            .send(body)
-        });
+    const allowForbiddenManifestChanges = req.body.forbidden_changes === undefined ? true : req.body.forbidden_changes;
+    const deploymentName = req.params.name;
 
-      });
+    function updateDeployment() {
+      return self.fabrik
+        .createOperation('update', {
+          deployment: deploymentName,
+          username: req.user.name,
+          arguments: req.body
+        }).invoke()
+        .then(body => {
+          res.format({
+            html: () => res
+              .redirect(303, redirect_uri),
+            default: () => res
+              .status(200)
+              .send(body)
+          });
+        });
+    }
+    logger.info(`Forbidden Manifest flag set to ${allowForbiddenManifestChanges}`);
+    if (allowForbiddenManifestChanges === 'false') {
+      logger.info(`Doing update if `);
```

Comment:
add a better log statement here.

Reply:
Log statement already present in tap below . removing.